package.json

```diff
@@ -59,7 +60,7 @@
     "classnames": "^2.2.5",
     "@algolia/events": "^4.0.1",
     "hogan.js": "^3.0.2",
-    "preact": "^10.0.0",
+    "preact": "^10.5.14",
```

Comment:
Why is [v10.5.14](https://github.com/preactjs/preact/releases/tag/10.5.14) required now?

Reply:
it was the latest version that vscode suggested, I didn't check whether it was actually the latest, but it required at least https://github.com/preactjs/preact/pull/3320

Reply:
This PR was released in [v10.6.0](https://github.com/preactjs/preact/releases/tag/10.6.0).

Reply:
will update to the latest!